src/Nox.Types.Abstractions/TypeOptions/NuidTypeOptions.cs

```diff
@@ -4,6 +4,7 @@ namespace Nox.Types;
 
 public class NuidTypeOptions
 {
+    public string? Prefix { get; set; }
```

Comment:
Add a comment, explaining the puposes, its optional, and and will be the prefix if this is not set